### PR TITLE
Improve animation support

### PR DESCRIPTION
**Pull request by cfsnook**
fixes some problems with unreliable starting 
improves toolbar and pop-up menus and removes unnecessary dependency on diagrams plugin

### CODE REVIEW

File: ac.soton.eventb.probsupport.sdk/feature.properties [hunk 1]
```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2020-2020 University of Southampton.
+# Copyright (c) 2020-2021 University of Southampton.
```
Comment (tshoang) [inline ac.soton.eventb.probsupport.sdk/feature.properties]: We should use the standard format 2020, 2021 for this

File: ac.soton.eventb.probsupport/OSGI-INF/l10n/bundle.properties [hunk 2]
```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-#  Copyright (c) 2020-2020 University of Southampton.
+#  Copyright (c) 2020-2021 University of Southampton.
```
Comment (tshoang) [inline ac.soton.eventb.probsupport/OSGI-INF/l10n/bundle.properties]: "2020, 2021"

File: ac.soton.eventb.probsupport.feature/feature.properties [hunk 3]
```diff
@@ -1,5 +1,5 @@
 ###############################################################################
-# Copyright (c) 2020-2020 University of Southampton.
+# Copyright (c) 2020-2021 University of Southampton.
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License v1.0
```
Comment (tshoang) [inline ac.soton.eventb.probsupport.feature/feature.properties]: EPL v2.0?

File: ac.soton.eventb.probsupport.sdk/feature.properties [hunk 4]
```diff
@@ -1,5 +1,5 @@
...
 # All rights reserved. This program and the accompanying materials
 # are made available under the terms of the Eclipse Public License v1.0
```
Comment (tshoang) [inline ac.soton.eventb.probsupport.sdk/feature.properties]: EPL v2.0?

**Review by tshoang — CHANGES_REQUESTED** [review]
I think we should use EPL v2.0 for consistency.

**Comment by cfsnook** [comment]
Thanks.. i think i have fixed all the copyrights now

File: ac.soton.eventb.probsupport.sdk/feature.properties (hunk 4 above)
Comment (cfsnook) [reply ac.soton.eventb.probsupport.sdk/feature.properties]: All fixed now

**Comment by cfsnook** [comment]
Ready to release. I have checked builds locally. Is there some releng to update?

**Review by tshoang — APPROVED** [review]
Looks good to me.

**Pull request merged**